### DISCUSSION TITLE
Icon browser not working

### DISCUSSION THREAD

**Discussion by boardshepherd** [flet-dev/flet#932] [Q&A] [answered]
Hi,

I needed to browse icons at [https://flet-icons-browser.fly.dev/#/](https://flet-icons-browser.fly.dev/#/) , but the service seem to be down. Just letting you know so it can be fixed.

Thanks

Matej Holly

**Comment by ndonkoHenri** [comment, marked as answer]
Thanks for the ping.
You can use https://flutils.fly.dev in the mean while if you need an icon browser.
